src/models/shipment.ts

```diff
@@ -83,4 +88,13 @@ export default class Shipment extends Model<
   @UpdatedAt
   @Column
   public readonly updatedAt!: Date
+
+  public displayName(): string {
+    return [
+      'Shipment',
+      this.shippingRoute,
+      this.labelYear,
+      this.labelMonth,
```

Comment:
Suggestion (non-blocking): pad with a `0` if it's a single-digit month so that it's consistent with the UI.
```suggestion
      this.labelMonth.toString().padStart(2, "0"),
```